cmd/gen-go-sample/inittree.go

```diff
@@ -232,6 +232,9 @@ func (t *initTree) Print(w io.Writer, g *generator) error {
 }
 
 func (t *initTree) print(w *bufio.Writer, g *generator, ind int) error {
+	if t.typ.prim != 0 && t.leafVal == "" {
+		return errors.E(nil, "init value not defined")
```

Comment:
"init value for primitive type not defined"?

Reply:
good catch, done!